exp/yaml_migrate/yaml_migrate.go

```diff
@@ -0,0 +1,58 @@
+package yaml_migrate
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+//ConvertToYaml method converts fogg.json to fogg.yml
+func ConvertToYaml() error {
+	jsonFile, err := os.Open("fogg.json")
```

Comment:
other commands let users override their path to the config file, should this command also allow for that?